doc/ext/traffic-server.py

```diff
@@ -440,6 +440,14 @@ def setup(app):
     app.add_crossref_type('configfile', 'file',
                           objname='Configuration file',
                           indextemplate='pair: %s; Configuration files')
+    
+    # Very ugly, but as of Sphinx 1.8 it must be done. There is an `override` option to add_crossref_type
+    # but it only applies to the directive, not the role (`file` in this case). If this isn't cleared
+    # explicitly the build will fail out due to the conflict. In this case, since the role action is the
+    # same in all cases, the output is correct. This does assume the config file names and log files
+    # names are disjoint sets.
+    del app.registry.domain_roles['std']['file']
+
```

Comment:
Could we rename our `:file:` role (`:configfile:` ?) so it no longer collides with the one in `std` domain?